src/interfaces/common.ts

```diff
@@ -11,17 +11,22 @@ export enum BooleanEnum {
 
 // All variables below were defined by DataPortal. Do not change their values as it will affect our metrics logs and dashboards.
 export enum AnalyticsEventTypes {
-	ScreenEvent = "screen",
-	UiEvent = "ui",
-	TrackEvent = "track",
-	OperationalEvent = "operational",
-	TraitEvent = "trait",
```

Comment:
Removed this as the trait event is primarily used by the personalization team (major use case is for in-product messaging and onboarding experiences). We have been advised by the analytics team to only use the above 4 event types.